appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java

```diff
@@ -521,8 +521,8 @@ private boolean isSimpleMode() {
         String columnID = "jobinstanceid";
         try (Connection connection = dataSource.getConnection()) {
             try (PreparedStatement preparedStatement = connection.prepareStatement(query)) {
-                preparedStatement.setString(1, parameter1);
-                preparedStatement.setString(2, parameter2);
+                preparedStatement.setInt(1, Integer.parseInt(parameter1));
```

Comment:
You will need to test this change with other database servers

Reply:
Tested with PostgreSQL, Derby and MySQL

Reply:
I tested it with Derby

Reply:
Also tested with H2

Reply:
Oracle DB is the last big'un to test against then